client/components/post-schedule/clock.jsx

```diff
@@ -38,17 +38,24 @@ function checkTimeValue( value ) {
 }
 
 class PostScheduleClock extends Component {
+	constructor( props ) {
+		super( props );
+
```

Comment:
One of the things I learned from @aduth is the following pattern that keeps the class dumb to the arguments that it's being passed if they're not needed. 

``` js
class PostScheduleClock extends Component {
    constructor() {
        super( ...arguments );
        // ...
    }

    // ...
}
```
It doesn't make a huge difference but I like it's simplicity